package.json

```diff
@@ -164,7 +164,7 @@
     "webpack": "^5.93.0"
   },
   "peerDependencies": {
-    "react": ">=17.0.0"
+    "react": ">=18.0.0"
```

Comment:
Good catch! I think we want to be react 17 friendly for a short while. 

Could you help us identify the hooks that rely on `useId` and help write a custom impl. that lives within our repo? (@Elvincth is helping us migrate away from @mantine/* so hopefully this shouldn't be a huge issue)

Reply:
I think a simpler approach would be to downgrade the version of @mantine/* to meet the requirements of react>=17. What do you think?

Reply:
Unfortunately, that is not an option, it'd require a lot more changes. Mantine v6 -> v7 was a breaking change. 😟